proposal.md

```diff
@@ -1,55 +1,57 @@
 # webRequest TLS Introspection API Extension Propsal
 
-This document outlines a proposed extension to the webRequest browser extension API that would add TLS introspection capabilities that are currently missing from all major browsers.
+This document outlines a proposed extension to the webRequest browser extension API that would add TLS introspection capabilities that are currently missing from all major browsers (exlucding the soon to be deprecated XPCOM/XUL Firefox APIs).
 
 ## The Problem
 
 While the existing webRequest API provides basically carte blanche access to the request life cycle the one thing it is missing is any form of introspection into the TLS connection state other than knowing that a user may have requested a HTTPS URL.
 
-The lack of any kind of introspection prevents the development of a number of security focused plugins such as:
+The lack of any kind of introspection prevents the development of a number of security focused browser extensions such as:
 
 a. plugins that feed third-party telemetry systems like EFF's DSO
```

Comment:
You `s/plugins/browser extensions/` above, but these all still say plugins :-) Since plugin is sort of a dirty word for web browsers, how about using "extensions" here.